htlcswitch/switch.go

```diff
@@ -105,6 +105,9 @@ type ChanClose struct {
 	// process for the cooperative closure transaction kicks off.
 	TargetFeePerKw chainfee.SatPerKWeight
 
+	// Optional delivery script to pay funds out to.
```

Comment:
Nit: godoc is usually of the form "DeliveryScript ...`, weird the linter didn't catch this.